docs/src/main/sphinx/security/file-system-access-control.rst

```diff
@@ -25,6 +25,12 @@ cluster in a single JSON file.
 Configuration
 -------------
 
+.. warning::
+
+    Access to :doc:`table functions </functions/table>` is allowed by default.
```

Comment:
Isnt it allowed for all functions?

Reply:
True, I wrote this warning because before access to table functions was always denied - there was no rules to control the access. Maybe change this into:
```
Access to all functions including table functions is allowed by default.
```
@mosabua WDYT?

Reply:
yes